src/Microsoft.ML.Transforms/Text/TextTransformCatalog.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML
```

Comment:
(nit) Since this is in the `Microsoft.ML` namespace, my preference would be for the file to not be under the `Text` folder. I think namespaces and folders should align.